ovn-fake-multinode-utils/playbooks/deploy-minimal.yml

```diff
@@ -34,7 +33,7 @@
     shell: |
       sed -i '/^MaxSessions /d' /etc/ssh/sshd_config
       echo "MaxSessions 1024" >> /etc/ssh/sshd_config
-      systemctl reload sshd.service
+      systemctl reload sshd.service || systemctl reload ssh.service || true
```

Comment:
Same nit as above about potentially using `ansible_os_family`.

Reply:
Alternatively, maybe systemd module can correctly ignore re-starts of socket-activated services?
I.e. maybe we will not need to try to restart it twice and || true, if we split the task in two, one that updates teh config and one that restarts the daemon with systemd module.

I didn't try that though.

Reply:
Yup, I ended up doing that, and apparently the `state: reloaded` gracefully works fine with a socket activated service